src/test/java/liuni/LiUniResourceTest.java

```diff
@@ -68,14 +66,16 @@ public void tearDown() {
     /* Test LiUniResource .fetchTimeline() REST method */
     @Test
     public void testREST_fetchTimeline_emptyTimeline() {
-        ResponseList<Status> respList = new ResponseListImpl<Status>();
+        List<TwitterTweetModel> tweetModelList = new ArrayList<TwitterTweetModel>();
         try {
-            doReturn(respList).when(twitter).getHomeTimeline();
+            TwitterService twitterService = mock(TwitterService.class);
+            when(twitterService.getTimeline()).thenReturn(tweetModelList);
+            resource.setTwitterService(twitterService);
 
             Response resp = resource.fetchTimeline();
 
             assertEquals(Response.Status.OK.getStatusCode(), resp.getStatus());
-            assertEquals(respList, resp.getEntity());
+            assertEquals(tweetModelList, resp.getEntity());
```

Comment:
what is the equals testing right here? asking in terms of java

Reply:
Done